### DISCUSSION TITLE
how to turn off command ./skript:~

### DISCUSSION THREAD

**Discussion by daechi12** [SkriptLang/Skript#5741] [General]
i want to turn off this feature. how can i do?
![image](https://github.com/SkriptLang/Skript/assets/136454408/8003547a-cebc-42ac-8aab-69ef558a4f59)

**Comment by TheLimeGlass** [comment]
Start using `on command`

**Reply by Fusezion** [reply]
False, he'd need player command map

**Reply by TheLimeGlass** [reply]
You can still use on command to avoid registering a command like how is presented.

**Reply by daechi12** [reply]
i want that players can't see all those.

**Reply by AyhamAl-Ali** [reply]
You can use either of above's solutions but for even better solution, use SkBee's tab completion feature and block tab completion for `/skript` command for non-authorized players. this way only authorized players can see the tab completions of /skript

**Reply by daechi12** [reply]
on tab complete:
	if event-string is "/skript":
		clear tab completions

so i use this code. but it dosen't  work. they  aren't hided

**Reply by sovdeeth** [reply]
> on tab complete:
> 	if event-string is "/skript":
> 		clear tab completions
> 
> so i use this code. but it dosen't  work. they  aren't hided

Tab complete is for arguments, not commands.
You want the command map.

**Reply by daechi12** [reply]
i didn't understand your words. sorry. can you edit my code, please?

**Comment by AyhamAl-Ali** [comment]
Please use [skUnity discord server](https://discord.gg/skript) instead of Github discussions for Skript help.

**Comment by harmfulladvocado** [comment]
on send command list:
	set command list to command list where [input does not contain ":"]
